hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/validators/CryptoTransferValidator.java

```diff
@@ -115,6 +115,12 @@ public void pureChecks(@NonNull final CryptoTransferTransactionBody op) throws P
             // Verify that one and only one of the two types of transfers (fungible or non-fungible) is present
             validateFalsePreCheck(!nonZeroFungibleValueFound && nftIds.isEmpty(), EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS);
             validateFalsePreCheck(nonZeroFungibleValueFound && !nftIds.isEmpty(), INVALID_ACCOUNT_AMOUNTS);
+
+            // Verify that if there is only one fungible transfer and no non-fungible transfers,
+            // the fungible transfer should not be empty
+            if (fungibleTransfers.size() == 1 && nftIds.isEmpty()) {
```

Comment:
@Neeharika-Sompalli  @mhess-swl  

Without a newly added check, this [test](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoTransferHandlerPureChecksTest.java#L289) was failing. Is it valid or we should edit the test and let the validator be as it was before?

Reply:
This check should've been taken care of by the lines directly above, shouldn't they @Neeharika-Sompalli?

i.e.
```
validateFalsePreCheck(!nonZeroFungibleValueFound && nftIds.isEmpty(), EMPTY_TOKEN_TRANSFER_ACCOUNT_AMOUNTS);
validateFalsePreCheck(nonZeroFungibleValueFound && !nftIds.isEmpty(), INVALID_ACCOUNT_AMOUNTS);
```

At the very least, thes new line should be `fungibleTransfers.size() > 0` instead of ` == 1`